modules/raft/src/test/java/org/apache/ignite/raft/jraft/rpc/IgniteRpcTest.java

```diff
@@ -107,6 +110,15 @@ public IgniteRpcTest(TestInfo testInfo) {
 
         service.start();
 
+        // Remove a node from Physical Topology as soon as it gets lost by SWIM.
+        InternalTopologyService internalTopologyService = (InternalTopologyService) service.topologyService();
```

Comment:
This is interesting, why did you have to explicitly add this to a test? Is there a way to have this listener "by default" when starting all necessary components?